watchtower/wtdb/client_db.go

```diff
@@ -94,6 +94,10 @@ var (
 	// 	db-session-id -> last-channel-close-height
 	cClosableSessionsBkt = []byte("client-closable-sessions-bucket")
 
+	// cTaskQueue is a top-level bucket where the disk queue may store its
+	// content.
+	cTaskQueue = []byte("client-task-queue")
```

Comment:
Note to self: this queue also needs to be in a namespace specific to its policy type since each different client instance requires its own queue. 

so either: own queue per policy type OR do one huge refactor so that the queue holds updates of all policy types & then splitting of which update goes where happens _after_.

Reply:
added this